packages/use-query-params/README.md

```diff
@@ -86,3 +86,28 @@ const Component = () => {
   )
 }
 ```
+
+### Push onto the stack instead of replacing
+
+To avoid mutating history
+
+```js
+// In this exemple we assume that we have an URL that include : `?company=Scaleway".
+import React from 'react'
+import useQueryParams from '@scaleway/use-query-params'
+
+const Component = () => {
+  const { queryParams, setQueryParams } = useQueryParams()
```

Comment:
```suggestion
  const { queryParams, replaceQueryParams } = useQueryParams()
```

Reply:
We can use both :)

Reply:
example use `replaceQueryParams` just below

Reply:
Right, sorry my bad :)